api/users.rb

```diff
@@ -90,26 +90,33 @@
 
     user_stats = {}
     thread_ids = {}
+    flat_thread_ids = []
 
     content_selector = {course_id: course_id, anonymous: false, anonymous_to_peers: false}
     if end_date
       content_selector[:created_at.lte] = end_date
     end
 
-    if user_id != '*'
-      content_selector["author_id"] = user_id
+    def set_template_result(user_id, user_stats, thread_ids)
       user_stats[user_id] = {
-        "num_threads" => 0,
-        "num_comments" => 0,
-        "num_replies" => 0,
-        "num_upvotes" => 0,
-        "num_downvotes" => 0,
-        "num_flagged" => 0,
-        "num_comments_generated" => 0
+          "num_threads" => 0,
+          "num_comments" => 0,
+          "num_replies" => 0,
+          "num_upvotes" => 0,
+          "num_downvotes" => 0,
+          "num_flagged" => 0,
+          "num_comments_generated" => 0,
+          "num_thread_followers" => 0,
+          "num_threads_read" => 0,
```

Comment:
Minor nit: The new indentation seems to be inconsistent with the rest of the file (everything else seems to use two spaces).